src/main/java/us/tastybento/bskyblock/api/configuration/ISettings.java

```diff
@@ -30,10 +31,10 @@ default void saveSettings() throws IllegalAccessException, IllegalArgumentExcept
     }
 
     default void saveBackup() throws IllegalAccessException, IllegalArgumentException, InvocationTargetException, SecurityException, InstantiationException, NoSuchMethodException, IntrospectionException, SQLException {
-        // Save backup in real database
+        // Save backup 
         @SuppressWarnings("unchecked")
-        AbstractDatabaseHandler<T> dbhandler =  (AbstractDatabaseHandler<T>) BSBDatabase.getDatabase().getHandler(getInstance().getClass());
-        dbhandler.saveObject(getInstance());
+        AbstractDatabaseHandler<T> backupHandler =  (AbstractDatabaseHandler<T>) new FlatFileDatabase().getHandler(getInstance().getClass());
```

Comment:
This means we could also create some backups when performing a purge?

Reply:
This back up is currently unused and meant to be a copy of the config (and any other YAML file). It’s there so we can compare the old and the new. That still needs to be coded.